sonar-plugin-api/src/main/java/org/sonar/api/batch/PostJob.java

```diff
@@ -29,7 +29,9 @@
  * it is an asynchronous processing on server side.
  *
  * @since 1.10
+ * @deprecated since 5.6 use {@link org.sonar.api.batch.postjob.PostJob}
```

Comment:
Javadoc in Java 8 seems to be buggy and removes the link, so HTML output is simply "@deprecated since 5.6 use PostJob". This is misleading as the classname is the same. I suggest to simply remove the link and use the complete label `@deprecated since 5.6 use org.sonar.api.batch.postjob.PostJob`